test/unit/coffee/RoomManagerTests.coffee

```diff
@@ -182,16 +182,16 @@ describe 'RoomManager', ->
 					# second for the leave
 					@RoomManager._clientsInRoom
 						.withArgs(@client, @doc_id)
-						.onCall(0).returns(0)
-						.onCall(1).returns(0)
+						.onCall(0).yields(0)
+						.onCall(1).yields(0)
```

Comment:
Is it worth just saying `.yields` and dropping the `.onCall` calls, if we just want it to always yield 0?

Reply:
As the comment explains above, these stubs are for two explicit calls: join, then leave.
We could remove the per argument spec too, but this explicit subbing is great for documentation.